lib/mongoose_push/service/apns/supervisor.ex

```diff
@@ -72,4 +98,69 @@ defmodule MongoosePush.Service.APNS.Supervisor do
 
     [single_config | mode_list]
   end
+
+  defp convert_token_pool_to_sparrow({pool_name, pool_config}, {mode_list, tokens}) do
+    auth_type = {:auth_type, :token_based}
+    pool_size = {:worker_num, pool_config[:pool_size]}
+
+    endpoint_mode = @default_endpoints[pool_config[:mode]]
+    endpoint = {:endpoint, pool_config[:endpoint] || endpoint_mode}
+
+    port_config =
+      case pool_config[:use_2197] do
+        true ->
+          2197
+
+        _ ->
+          nil
+      end
+
+    port = {:port, port_config}
+
```

Comment:
Why do you need the new lines between `port` and `name` and not new lines between `key` and `team`??